packages/faucet/src/metrics.ts

```diff
@@ -0,0 +1,52 @@
+import { Logging } from '@google-cloud/logging'
```

Comment:
So of course we do not have any standardization so to speak, but I generally would prefer for us not to directly log to GCP. Instead, GCP already aggregates the logs for us from STDOUT/STDERR and deals with doing so in a performant manner, batch, etc.

It also avoids us having to configure the client below, give the appropriate labels like project/region, and many more useful information such as execution id, severity etc.

Reply:
In general, I agree. But this is different.
The reason I'm using this library is to create my own `LogEntry` for stackdriver.
See that I'm using:

 - A different resource (fantasy function name on the metadata)
 - structured log entry, not a string

See the screenshot of an actual log:
![image](https://user-images.githubusercontent.com/656276/65981126-d98d2a00-e44e-11e9-9b6e-a57c598157a6.png)

This allow to easily create new metric with metric labels using structured fields.